symphony-bdk-spring/bdk-core-starter/src/main/java/com/symphony/bdk/spring/config/BdkApiClientsConfig.java

```diff
@@ -0,0 +1,185 @@
+package com.symphony.bdk.spring.config;
+
+import com.symphony.bdk.core.client.ApiClientFactory;
+import com.symphony.bdk.gen.api.AppEntitlementApi;
+import com.symphony.bdk.gen.api.ApplicationApi;
+import com.symphony.bdk.gen.api.AttachmentsApi;
+import com.symphony.bdk.gen.api.AuditTrailApi;
+import com.symphony.bdk.gen.api.CertificatePodApi;
+import com.symphony.bdk.gen.api.ConnectionApi;
+import com.symphony.bdk.gen.api.DatafeedApi;
+import com.symphony.bdk.gen.api.DefaultApi;
+import com.symphony.bdk.gen.api.DisclaimerApi;
+import com.symphony.bdk.gen.api.DlpPoliciesAndDictionaryManagementApi;
+import com.symphony.bdk.gen.api.InfoBarriersApi;
+import com.symphony.bdk.gen.api.MessageApi;
+import com.symphony.bdk.gen.api.MessagesApi;
+import com.symphony.bdk.gen.api.PodApi;
+import com.symphony.bdk.gen.api.PresenceApi;
+import com.symphony.bdk.gen.api.RoomMembershipApi;
+import com.symphony.bdk.gen.api.SecurityApi;
+import com.symphony.bdk.gen.api.SessionApi;
+import com.symphony.bdk.gen.api.ShareApi;
+import com.symphony.bdk.gen.api.SignalsApi;
+import com.symphony.bdk.gen.api.StreamsApi;
+import com.symphony.bdk.gen.api.SystemApi;
+import com.symphony.bdk.gen.api.UserApi;
+import com.symphony.bdk.gen.api.UsersApi;
+import com.symphony.bdk.gen.api.UtilApi;
+import com.symphony.bdk.gen.api.ViolationsApi;
+
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Injection of all available raw-APIs generated from the official Symphony Swagger specifications.
+ *
+ * <p>
+ *   Please note that each raw-API requires a particular {@link com.symphony.bdk.core.api.invoker.ApiClient} as unique
+ *   constructor parameter, depending on which component it targets:
+ *   <ul>
+ *     <li>Pod</li>
+ *     <li>Agent</li>
+ *   </ul>
+ * </p>
+ *
+ * <p>
+ *   Those beans are not intended to be used by end-developers. They are injected into the Spring application context
```

Comment:
In that case do we want to make them bean and put them in the context? (vs creating them manually where needed, i.e in BdkServiceConfig

Reply:
I think that : "just in case", I'd feel more confident to have them injected in the context. 

Actually, I already imagine a customer request saying: "oh we need to call this API (such as DLP, Security...) but the service does not already exists". In this situation we could easily reply by : OK you just have to inject this bean and this bean and you'll be able to proceed to the API call, the service layer will come later. 

Maybe I should refine this Javadoc comment

Reply:
ok, I'll guess we won't document that they exist then